gcc error when building

I'm attempting to build hail from a clone of this repository's master branch, as a local install on my laptop, under Debian GNU/Linux version 8.  The gradle script successfully downloaded and installed the various Java dependencies, but gcc chokes on the C source code.  I get:
$ ./gradlew shadowJar
:compileJava UP-TO-DATE
:nativeLib
(cd libsimdpp-2.0-rc2 && cmake .)
-- Configuring done
-- Generating done
-- Build files have been written to: /home/rmk/package_sources/hail/hail/src/main/c/libsimdpp-2.0-rc2
mkdir -p lib/linux-x86-64
g++ -fvisibility=hidden -rdynamic -shared -fPIC -ggdb -O3 -march=native -g -std=c++11 -Ilibsimdpp-2.0-rc2 -Wall -Werror  ibs.cpp -o lib/linux-x86-64/libibs.so
In file included from ibs.cpp:1:0:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/popcntintrin.h: In function ‘uint64_t vector_popcnt(uint64vector)’:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/popcntintrin.h:42:1: error: inlining failed in call to always_inline ‘long long int _mm_popcnt_u64(long long unsigned int)’: target specific option mismatch
 _mm_popcnt_u64 (unsigned long long __X)
 ^
ibs.cpp:14:48: error: called from here
   uint64_t count = _mm_popcnt_u64(extract<0>(x));
                                                ^
In file included from ibs.cpp:1:0:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/popcntintrin.h:42:1: error: inlining failed in call to always_inline ‘long long int _mm_popcnt_u64(long long unsigned int)’: target specific option mismatch
 _mm_popcnt_u64 (unsigned long long __X)
 ^
ibs.cpp:16:41: error: called from here
   count += _mm_popcnt_u64(extract<1>(x));
                                         ^
make: *** [lib/linux-x86-64/libibs.so] Error 1
Makefile:52: recipe for target 'lib/linux-x86-64/libibs.so' failed
:nativeLib FAILED

Tim Poterba suggested defining `CXXFLAGS='-DHAIL_OVERRIDE_ARCH -DSIMDPP_ARCH_X86_SSE2'`, but to no avail.  So, he suggested I open this issue.

I'm running gcc version 4.9.2.  Possibly relevant might be the processor I'm running,
$ lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                2
On-line CPU(s) list:   0,1
Thread(s) per core:    1
Core(s) per socket:    2
Socket(s):             1
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 23
Model name:            Intel(R) Core(TM)2 CPU         P8600  @ 2.40GHz
Stepping:              10
CPU MHz:               800.000
CPU max MHz:           2401.0000
CPU min MHz:           800.0000
BogoMIPS:              4800.16
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              3072K
NUMA node0 CPU(s):     0,1

--- a/src/main/c/ibs.cpp
+++ b/src/main/c/ibs.cpp
@@ -1,4 +1,3 @@
-#include <popcntintrin.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -11,15 +10,15 @@ using namespace simdpp;
 
 uint64_t vector_popcnt(uint64vector x) {
   #if UINT64_VECTOR_SIZE < 4
-  uint64_t count = _mm_popcnt_u64(extract<0>(x));
+  uint64_t count = __builtin_popcountll(extract<0>(x));
   #if UINT64_VECTOR_SIZE >= 2
-  count += _mm_popcnt_u64(extract<1>(x));
+  count += __builtin_popcountll(extract<1>(x));
   #endif
   #elif UINT64_VECTOR_SIZE == 4
-  uint64_t count = _mm_popcnt_u64(_mm256_extract_epi64(x.operator __m256i(), 0));
-  count += _mm_popcnt_u64(_mm256_extract_epi64(x.operator __m256i(), 1));
-  count += _mm_popcnt_u64(_mm256_extract_epi64(x.operator __m256i(), 2));
-  count += _mm_popcnt_u64(_mm256_extract_epi64(x.operator __m256i(), 3));
+  uint64_t count = __builtin_popcountll(_mm256_extract_epi64(x.operator __m256i(), 0));
+  count += __builtin_popcountll(_mm256_extract_epi64(x.operator __m256i(), 1));
+  count += __builtin_popcountll(_mm256_extract_epi64(x.operator __m256i(), 2));
+  count += __builtin_popcountll(_mm256_extract_epi64(x.operator __m256i(), 3));
   #else
   #error "we do not support vectors longer than 4, please file an issue"
   #endif

